spec.md

```diff
@@ -145,28 +155,21 @@ the value is an RGBA color (4 byte, `0-255` per channel) for representing the ob
 
 The `image` key is an optional dictionary which contains information on the image the label is associated with.
 If included it must include a key `array` whose value that is either:
```

Comment:
Minor RFE, given the removal of the second paragraph, `either` is unnecessary.

Reply:
Also gone in the next set of commits.